DFC.ServiceTaxonomy.TestSuite/Features/Editor/Events/ContentChangeEvents_ExistingDraft.feature

```diff
@@ -0,0 +1,91 @@
+﻿@webtest
+Feature: ContentChangeEvent for existing draft
+
+Background: 
+	Given I logon to the editor
+	Given I set the content type to be "SharedContent" 
+	Given I set up a data prefix for "Title"
+
+	#Given the test is tagged with ""
+	And I Navigate to "/Admin/Contents/ContentTypes/SharedContent/Create" 
+	#And I have ensured the activity I intend to add doesn't exist
+	And I capture the generated URI
+	And I Enter the following form data for "SharedContent"
+	| Title              |  Content          |
+	| New Shared Content |  <p>Here it is<p> |
+	And I save the draft item
+	#Then an event of type "draft" has been issued to notify consumers of the change
+	#Given I check time of the latest event message
+	Given I check the number of events sent for this contentItem
+
+Scenario: 13. An update to an existing draft document is succesful
+	Given I Navigate to "/Admin/Contents/ContentItems" 
+	And I search for the "Title"
+	And I select the first item that is found
+	And I Enter the following form data for "SharedContent"
+	| Title                  | Content              |
+	| updated Shared Content | <p>Here it is now<p> |
+	When I save the draft item
+	Then the save action completes succesfully
+	And an event of type "Draft" has been issued to notify consumers of the change
+
```

Comment:
It'd be nice if the tests could check that no other events are published (with a list of know false-positives, until we get them fixed). Perhaps something like...
And no other events are published, except for these known false-positives
```
| EventType |
| draft     |
```
Any other events would fail the test.

Reply:
Agreed.. I have down as a todo

Reply:
Just noticed it 😄.
I was also going to suggest checking the fields in the event, but there's a todo for that too.

Reply:
I will be adopting your suggestion on layout :-)